### PR TITLE
Отвалилось левое меню

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

File: src/blocks/common.blocks/menu-list/_type/menu-list_type_main.bemtree [hunk 1]
```diff
@@ -44,12 +44,27 @@ block('menu-list').mod('type', 'main')(
...
     mode('item')(function() {
-        c = this.ctx;
-        return {
-            elem: 'link',
-            title: c.title,
-            url: c.url,
-            elemMods: { main: true, active: c.active, type: c.type, size: c.size }
-        };
+        var ctx = this.ctx,
+            result = {
```
Comment (tadatuta) [inline src/blocks/common.blocks/menu-list/_type/menu-list_type_main.bemtree:48]:
``` js
return ctx.type === 'delimeter' ? { elem: 'delimeter'} : {
    // ...
}
```

Comment (tavriaforever) [inline src/blocks/common.blocks/menu-list/_type/menu-list_type_main.bemtree:48]: Looks much better, we improve it

Comment (gela-d) [inline src/blocks/common.blocks/menu-list/_type/menu-list_type_main.bemtree:48]: @tadatuta @tavriaforever https://github.com/bem/bem-site-engine/pull/45

**Pull request merged**